system/core/document.php

```diff
@@ -257,7 +256,7 @@ class HTMLForm {
 
 			Parameters:
 			$form - A DOMWrap\Element that contains the form
-			        fields.
+							fields.
```

Comment:
Here, the indentation was correct,  but is now no longer. This is is a bit of a special line, since it consists of two types of indentation: First a number tabs that indent the entire block. These tabas should match the preceding lines. Then, there should be a number of spaces, to make sure that "fields" lines up with "A DOMWrap". This second type should be spaces rather than tabs, since proper alignment depends on the tab width used for display (unlike the first type of indentation, where the only alignment needed is with other lines also indented by tabs, so you can change the tab display width to change the look, without messing up any alignment).

This is one of the reasons I often use space-only indentation in my code, but with some consideration, this tab-based (mixed with spaces in cases like this) can also work fine.